setup.py

```diff
@@ -30,6 +30,38 @@ def package_data(pkg, roots):
     return {pkg: data}
 
 
+def is_requirement(line):
+    """
+    Return True if the requirement line is a package requirement;
+    that is, it is not blank, a comment, or editable.
+    """
+    # Remove whitespace at the start/end of the line
+    line = line.strip()
+
+    # Skip blank lines, comments, and editable installs
+    return not (
```

Comment:
Loading setup.py requirements from a file was tripping up tox, so I ditched it for now, in the interest of being able to redeploy everything sooner rather than later.